docs/betaseries.rst

```diff
@@ -122,39 +117,39 @@ more reliable.
    for event, beta in zip(events, betas):
        data = X[event] * beta + error
 
-This python psuedocode demonstrates LSS where each event
-is given it's own model.
+This python pseudocode demonstrates LSS where each event
+is given its own model.
 
-Relationship to Resting State
+Relationship to Resting-State Functional Connectivity
 -----------------------------
-Betaseries is similar to resting state because the same analyses
-applied to resting state data can ostensibly be applied to betaseries.
-At the core of both resting state and betaseries we are working with
-a list of numbers at each voxel.
+Betaseries is similar to resting-state functional connectivity (time-series correlations) because the same analyses
+typically applied to resting-state data can ostensibly be applied to betaseries.
+At the core of both resting-state functional connectivity and betaseries we are working with
+a vector of numbers at each voxel.
```

Comment:
just curious if it more common to say "vector" as opposed to "list". My intuition is that "vector" is more common term in statistics/mathematics, but "list" is closer to the programmatic structure the numbers will be stored in. 

I believe I agree with the change, but I would like to hear your thoughts so I can understand this better.

Reply:
Yeah that's a good point, I think leaving it as list is totally fine in keeping with the more software-oriented interpretation! Consider my edits more "suggestions" than strictly necessary changes.

Reply:
I'll accept vector since this section is more about the statistical concepts than the programmatic implementation (it could just as easily be a tuple or some other data structure)